core/system/src/mpu/vmpu_armv7m.c

```diff
@@ -146,60 +58,61 @@ static const TMpuRegion* vmpu_fault_find_region(uint32_t fault_addr)
 
 uint32_t vmpu_fault_find_acl(uint32_t fault_addr, uint32_t size)
 {
-    const TMpuRegion *region;
+    const MpuRegion *region;
 
     /* return ACL if available */
-    switch (fault_addr) {
-        case (uint32_t) &SCB->SCR:
-            return UVISOR_TACL_UWRITE | UVISOR_TACL_UREAD;
-        default:
-            /* translate fault_addr into its physical address if it is in the bit-banding region */
-            if (fault_addr >= VMPU_PERIPH_BITBAND_START && fault_addr <= VMPU_PERIPH_BITBAND_END) {
-                fault_addr = VMPU_PERIPH_BITBAND_ALIAS_TO_ADDR(fault_addr);
-            }
-            else if (fault_addr >= VMPU_SRAM_BITBAND_START && fault_addr <= VMPU_SRAM_BITBAND_END) {
-                fault_addr = VMPU_SRAM_BITBAND_ALIAS_TO_ADDR(fault_addr);
-            }
+    if (fault_addr == (uint32_t) &SCB->SCR) {
```

Comment:
@niklas-arm @AlessandroA SCB->SCR access is strictly speaking a stability-backdoor. We must change that into a RLA gateway in mbed-os as we have it now. Please add a fat fixme here.